esp8266-weather-station-color.ino

```diff
@@ -238,15 +260,38 @@ uint8_t currentTouchPoint = 0;
 void loop() {
   static bool asleep = false; //  asleep used to stop screen change after touch for wake-up
   gfx.fillBuffer(MINI_BLACK);
-  if (touchController.isTouched(0)) {
+
+  /* Break up the screen into 4 sections a touch in section:
```

Comment:
I think it would finally be time to document the touch behavior in the README 😉 Even the 12/24h switching was documented - shame on us.

Reply:
I updated the README for the force calibration, touch screen points and added an image